### ISSUE TITLE
TypeError: can't subtract offset-naive and offset-aware datetimes in wrapper.py _setTimer()

### ISSUE THREAD

**Issue by mrwillett** [erdewit/ib_insync#449]
Hi Ewald,

I've been using your (wonderful) library for several months now, and I'd like to help contribute. I've found that when using Watchdog to run the connection with IB Gateway through IBC, a non-critical warning can be seen in the logs after a warning is triggered from IB's servers.

I have only been able to reproduce the error with Watchdog, not while running IBC by itself. The method I used to quickly force a warning from IB was to spam requests to achieve the ```Warning: Approaching max rate of 50 messages per second``` warning. This would invariably be followed immediately in the logs with:
```py
ERROR:base_events:default_exception_handler:line 1738:Exception in callback Wrapper._setTimer()
handle: <TimerHandle when=57.594543062 Wrapper._setTimer()>
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/mike/.venv/ibenv/lib/python3.9/site-packages/ib_insync/wrapper.py", line 213, in _setTimer
    diff = (now - self.lastTime).total_seconds()
TypeError: can't subtract offset-naive and offset-aware datetimes
```
The solution I found was to modify the _setTimer() method of Wrapper with:

```diff
- diff = (now - self.lastTime).total_seconds()
+ diff = (now - self.lastTime.astimezone(timezone.utc)).total_seconds()
```

Let me know if you want more detailed logs or a minimum reproducible script, or if you are aware of this behavior and there's a specific reason for leaving diff as-is. Otherwise, I'll submit a PR with the change.

Thanks,

Michael

**Comment by erdewit** [comment]
Thank you for the bug report. Now ``wrapper.lastTime`` should be a timezone-aware datetime, and if the timezone is missing then that is a deeper bug by itself. After going through the code I can see no way how the timezone can be missing though, it is always explicitly set to be UTC. If you can provide a minimum case to reproduce the issue that would be appreciated.

**Comment by mrwillett** [comment]
You are totally right, there is no bug on your end. I made a series of errors on my end. Long story short, I forgot that I had modified ```self.lastTime``` within the ```Wrapper.tcdDataArrtived()``` method so that it would return the time in EST, and I also made it timezone-unaware with ```.replace(tzinfo=None)```, which explains where the unaware timezone came from. Now I know to run ```git diff``` between the copy I'm using and a fresh copy before jumping to conclusions. I apologize for the hassle. I'll close the PR I made.
